### ISSUE TITLE
Applying filters doesn't communicate any changes to screen reader users

### ISSUE THREAD

**Issue by greatislander** [inclusive-design/covid-data-monitor#20]
## Describe the bug

When a filter is applied to the map, no information about the impact of the filter is communicated to screen reader users.

### To reproduce

1. Visit the map while using a screen reader.
2. Check a filter, e.g. 'Accessible Entrances'.
3. Click the 'Apply Filters' button.
4. Observe that some of the points on the map have dimmed, but nothing is announced via the screen reader.

### Expected behavior

As the impact of the filtering action is only evident to sighted users, there should be some means of announcing that the list/collection of matched testing sites has been updated to screen reader users, likely a visually-hidden [live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions). The content of the live region could be changed to "Filters applied — 73 testing sites matched." or something similarly 

## Screenshots

N/A.

### Desktop
 - OS: macOS 11.4 with VoiceOver
 - Browsers: Edge 91.0.864.48, Firefox 89.0, and Safari 14.1.1

### Smartphone / tablet 

N/A.

## Additional context or notes

N/A.

**Comment by BlueSlug** [comment]
This issue was also observed in Chrome 91 on Windows 10 Pro v20H2 using NVDA 2019.3.1:

![2021-06-14 wecount a11y map apply filter not announced](https://user-images.githubusercontent.com/547933/121933637-51c77c00-cd14-11eb-9ec2-509dae9ae164.png)

**Comment by uttaraghodke** [comment]
![image](https://user-images.githubusercontent.com/66276957/122791048-de2eed00-d286-11eb-9c6c-77d673db8ee9.png)

![image](https://user-images.githubusercontent.com/66276957/122791231-0cacc800-d287-11eb-92f9-b9338d6d6567.png)
